modules/core/src/com/haulmont/cuba/core/app/Emailer.java

```diff
@@ -138,23 +139,37 @@ protected void prepareEmailInfo(EmailInfo emailInfo) {
         }
     }
 
+    protected void processCaptionTemplate(EmailInfo emailInfo) {
+        String caption = buildStringWithTemplateParams(emailInfo, emailInfo.getCaption());
```

Comment:
This call can break valid caption if template params are empty in case of caption contains FreeMarker reserved symbols.
Probably we should add test for this case